bundles/org.openhab.ui/web/src/pages/home.vue

```diff
@@ -156,7 +156,7 @@ export default {
       this.$store.dispatch('stopTrackingStates')
     },
     onPageInit () {
-      this.$f7.panel.get('left').enableVisibleBreakpoint()
+      // this.$f7.panel.get('left').enableVisibleBreakpoint()
```

Comment:
not sure how you and the OH team usually handle this, but I think with version control it's not required to keep this line as a comment 😉